R/fit.R

```diff
@@ -241,13 +241,11 @@ refit_multiple_optimizers <- function(fit,
 #'   of the SAS results.
 #'
 #' - The argument `start` is used to facilitate the choice of initial values for fitting the model.
-#'   If `string` is provided, `mmrm_control` will try to find a function within the parent frame.
 #'   If `function` is provided, make sure its parameter is a valid element of `mmrm_tmb_data`
-#'   or `mmrm_tmb_formula_parts`.
-#'   If `NULL` is provided, `default_start` will be used.
-#'   By default, `default_start` will be used. Other implemented methods include `std_start` and
-#'   `emp_start`. `default_start` will use `emp_start` if covariance structure is unstructured and use
-#'   `std_start` otherwise.
+#'   or `mmrm_tmb_formula_parts` and it returns a numeric vector.
+#'   If `NULL` is provided, `std_start` will be used.
+#'   By default, `std_start` will be used. Other implemented methods include `std_start` and
```

Comment:
```suggestion
#'   By default and if `NULL` is provided, `std_start` will be used.
#'   Other implemented methods include `std_start` and
```
(just a bit shorter)